src/components/Image/docs/index.story.tsx

```diff
@@ -72,6 +66,27 @@ export default {
       tab({
         title: 'Usage',
         sections: [
+          description(
+            '`Image` is a component to literally display an image - whether an absolute with full URL or a media platform item with relative URI.',
+          ),
+
+          description({
+            text: [
+              <p style={{ marginBottom: '20px' }} key={`${dataHook}-links`}>
+                ⚠️ This component is mainly for general image usages - consider
+                using{' '}
+                <LinkTo kind={'Working In Progress'} story="HeroImage">
+                  {'<HeroImage/>'}
+                </LinkTo>{' '}
+                and{' '}
+                <LinkTo kind={'Working In Progress'} story="ThumbnailImage">
+                  {'<ThumbnailImage/>'}
+                </LinkTo>{' '}
+                instead, which are meant for common TPAs use-cases.
+              </p>,
```

Comment:
If we want that our users will use only the HeroImage and the ThumbnailImage so maybe we shouldn't expose the Image component. I mean maybe we could remove this story page.
WDYT?

Reply:
We thought about this concern if to make it internal, but at the moment we don't really know. In some cases they might want to use the image for different specific use cases so we decided to open it in the meantime